Jenkinsfile

```diff
@@ -328,7 +328,7 @@ def alertTestOutcome(params, testStatus) {
   }
 
   if (testStatus.mainFailed) {
-    def guideUrl = "https://github.com/alphagov/publishing-e2e-tests/blob/master/CONTRIBUTING.md#dealing-with-flaky-tests"
+    def guideUrl = "https://github.com/alphagov/publishing-e2e-tests/blob/main/CONTRIBUTING.md#dealing-with-flaky-tests"
```

Comment:
This URL is broken – but I'm assuming you'll rename the git branch after merging this PR, so this will be resolved.

Reply:
Yup 👍